data/json/recipes/recipe_others.json

```diff
@@ -3440,6 +3440,84 @@
     ]
   ]
 },{
+    "type" : "recipe",
+    "result": "bot_manhack_acid",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_OTHER",
+    "skill_used": "mechanics",
+    "skills_required": [[ "electronics", 5 ], [ "computer", 5 ]],
+    "difficulty": 7,
+    "time": 8000,
+    "reversible": true,
+    "decomp_learn": 7,
+    "autolearn": false,
+    "book_learn": [[ "recipe_lab_elec", 6 ] , [ "textbook_robots", 7]],
+    "qualities" : [
+      {"id":"SCREW","level":1,"amount":1}
+    ], "tools": [
+      [
+      [ "soldering_iron", 10 ],
+      [ "toolset", 10 ]
+      ]
+      ],
+    "components": [
+      [
+        [ "acidbomb", 1 ]
```

Comment:
It spreads way more acid than acidbomb does. Needs more acidbombs or different components - for example, more concentrated acid and a bigger bottle than the smallest one used for acidbomb.

Also, it needs something to detonate it, to spread the acid around. Can be gasoline/diesel/gunpowder/blackpowder, even just tiny amounts (50 charges of gasoline is tiny).